tests/workflows/test_pytorch_optuna.py

```diff
@@ -3,25 +3,30 @@
 import tempfile
 import zipfile
 
-import optuna
 import pytest
-import torch
-import torch.nn as nn
-import torch.nn.parallel
-import torch.optim as optim
-import torch.utils.data
-from dask.distributed import PipInstall, wait
+from dask.distributed import PipInstall
+
+from ..utils_test import wait
+
+optuna = pytest.importorskip("optuna")
 
 
 @pytest.mark.client(
     "pytorch_optuna",
     worker_plugin=PipInstall(
         [
+            # FIXME package_sync doesn't seem to deduce GPU specific installs of
+            #       libraries like torch
```

Comment:
@milesgranger @ntabris @shughes-uk is there a coiled issue for this we can reference here?

Reply:
https://github.com/coiled/platform/issues/1249